src/drift.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+/**
+ * Builds a normalized drift summary object from raw drift detection output.
+ *
+ * @param {*} results - Full drift detection results. Accepted for API compatibility,
```

Comment:
Why are the results being passed in if we determine drift from changes?

Reply:
Good catch, I fixed this issue